### ISSUE TITLE
Slow import library suggestion on IntelliJ 2021.1+

### ISSUE THREAD

**Issue by kefasjw** [dart-lang/sdk#48407]
Moved from https://github.com/flutter/flutter-intellij/issues/5969 as suggested

## Description

Slow import library suggestion on IntelliJ 2021.1 (and higher). It's instant on IntelliJ 2020.3 (and lower).

## Steps to Reproduce

1. Create a new flutter project
2. Wait for the initializer and analyzer to finish
3. Create a new dart file inside the `lib` directory
4. Copy paste this code into the file
    ```
    class MyWidget extends StatelessWidget {
      const MyWidget({Key? key}) : super(key: key);
    
      @override
      Widget build(BuildContext context) {
        return Container();
      }
    }
    ```
5. Click the `StatelessWidget` and press `ALT` + `Enter`

### Actual result

Pop up `Inspection 'Annotator' options` will be shown instead of import library suggestion. The import library suggestion will be shown after 2-3 times (2-5 seconds) after closing this pop up.

![image](https://user-images.githubusercontent.com/11681703/153743956-c73b1771-b5ec-47d2-8fba-c2781e81fea5.png)

### Expected result

Import library suggestion should be shown immediately

![image](https://user-images.githubusercontent.com/11681703/153744109-9dfa2755-7529-4ea6-8026-1f577201b704.png)

### 2021.1 (slow suggestion)
https://user-images.githubusercontent.com/11681703/153744299-b84426a6-79c2-4f0f-a179-e53278b0d6ae.mp4

### 2020.3 (instant suggestion)
https://user-images.githubusercontent.com/11681703/153745068-8b6030f3-fb8e-448a-92fa-eda7ca9d581f.mp4

## Version info

- Windows 11 (video), but also tested on Ubuntu 20.04 and macOS Monterey 12.2 with the same result

### Intellij CE 20.03 
- Dart Plugin = 203.8452
- Flutter Plugin = 64.1.1

### Intellij CE 2021.1
- Dart plugin 211.7798
- Flutter plugin 64.1.2

### Intellij CE 2021.3.2
- Dart plugin: 213.5744.122
- Flutter plugin: 64.1.4

### Intellij CE 2022.1 EAP (221.4164.146)
- Dart plugin: 221.4165.162
- Flutter plugin 65.0.6-dev.1

- Flutter doctor
    <details><summary>Show Flutter doctor</summary>
    <p>

    ```
    C:\Users\xxx>flutter doctor -v
    [√] Flutter (Channel stable, 2.10.1, on Microsoft Windows [Version 10.0.22000.469], locale en-ID)
        • Flutter version 2.10.1 at C:\Users\xxx\Downloads\test\flutter_windows_2.10.1-stable\flutter
        • Upstream repository https://github.com/flutter/flutter.git
        • Framework revision db747aa133 (3 days ago), 2022-02-09 13:57:35 -0600
        • Engine revision ab46186b24
        • Dart version 2.16.1
        • DevTools version 2.9.2
    
    [X] Android toolchain - develop for Android devices
        X Unable to locate Android SDK.
          Install Android Studio from: https://developer.android.com/studio/index.html
          On first launch it will assist you in installing the Android SDK components.
          (or visit https://flutter.dev/docs/get-started/install/windows#android-setup for detailed instructions).
          If the Android SDK has been installed to a custom location, please use
          `flutter config --android-sdk` to update to that location.
    
    
    [√] Chrome - develop for the web
        • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe
    
    [X] Visual Studio - develop for Windows
        X Visual Studio not installed; this is necessary for Windows development.
          Download at https://visualstudio.microsoft.com/downloads/.
          Please install the "Desktop development with C++" workload, including all of its default components
    
    [!] Android Studio (not installed)
        • Android Studio not found; download from https://developer.android.com/studio/index.html
          (or visit https://flutter.dev/docs/get-started/install/windows#android-setup for detailed instructions).
    
    [√] IntelliJ IDEA Community Edition (version 2020.3)
        • IntelliJ at C:\Users\xxx\Downloads\test\ideaIC-2020.3.4.win
        • Flutter plugin version 64.1.1
        • Dart plugin version 203.8452
    
    [√] IntelliJ IDEA Community Edition (version 2021.1)
        • IntelliJ at C:\Users\xxx\Downloads\test\ideaIC-2021.1.3.win
        • Flutter plugin version 64.1.2
        • Dart plugin version 211.7798
    
    [√] Connected device (3 available)
        • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.22000.469]
        • Chrome (web)      • chrome  • web-javascript • Google Chrome 98.0.4758.80
        • Edge (web)        • edge    • web-javascript • Microsoft Edge 98.0.1108.43
    
    [√] HTTP Host Availability
        • All required HTTP hosts are available
    
    ! Doctor found issues in 3 categories.
    ```
    </p>
    </details>

**Comment by srawlins** [comment]
Thanks for the detailed repro!

**Comment by ramyhq** [comment]
I have the same issue on android studio, please any help?
did you fix it?

**Comment by kefasjw** [comment]
This has been fixed in 2023.1 https://youtrack.jetbrains.com/issue/WEB-55217/Dart-quick-fixes-sometimes-dont-show-up-on-first-AltEnter-need-to-press-Esc-and-AltEnter-once-again
